### PR TITLE
minor doc improvement

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: templates/tfengine/components/cicd/README.md [hunk 1]
```diff
@@ -52,12 +52,19 @@ to detect changes in the repo, trigger builds, and run the workloads.
...
-    manually. To do so, add the `devops` root module (that hosts the devops
-    project) in the `managed_modules` list in the CICD Terraform Engine config.
-    Changes outside the approved set will still need to be made manually.
+    **NOTE**: The CICD service account can manage a subset of resources (e.g.
+    APIs) within its own project (`devop` project). This allows users to have
+    low risk changes made in the `devops` project deployed through the standard
+    Cloud Build pipeline, without needing to apply it manually. To do so, add
+    the `devops` module (that hosts the devops project) in the `managed_modules`
+    list in the CICD Terraform Engine config block. It is useful when you want
+    CICD to start managing a new API/service in other projects. (For the CICD
```
Comment (umairidris) [inline templates/tfengine/components/cicd/README.md:61]: This sentence seems redundant given the first sentence: The CICD service account can manage a subset of resources (e.g. **APIs**) within its own project (`devop` project).

File: templates/tfengine/components/cicd/README.md [hunk 2]
```diff
@@ -52,12 +52,19 @@ to detect changes in the repo, trigger builds, and run the workloads.
...
+    service account to manage services and resources, APIs of all resources
```
Comment (umairidris) [inline templates/tfengine/components/cicd/README.md:62]: Consider making this a separate note

File: templates/tfengine/components/cicd/README.md (hunk 1 above)
Comment (xingao267) [reply templates/tfengine/components/cicd/README.md]: Reworded and treat API as a typical use case for this feature. PTAL.

File: templates/tfengine/components/cicd/README.md [hunk 3]
```diff
@@ -52,12 +52,22 @@ to detect changes in the repo, trigger builds, and run the workloads.
...
     they are run.
 
-    NOTE: The CICD service account can manage a subset of resources (e.g. APIs)
-    within its own project. This allows users to have changes deployed for the
-    CICD through the standard Cloud Build pipeline, without needing to apply it
-    manually. To do so, add the `devops` root module (that hosts the devops
-    project) in the `managed_modules` list in the CICD Terraform Engine config.
-    Changes outside the approved set will still need to be made manually.
+    **NOTE**: The CICD service account can manage a subset of resources (e.g.
+    APIs) within its own project (`devop` project). This allows users to have
```
[the remark this answers is not in the document]
Comment (xingao267) [reply templates/tfengine/components/cicd/README.md]: Done

**Pull request merged**